bin/sirius-script-rabpm-mngr.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+import argparse
+import subprocess
+
+
+def execute_ssh(hostname, command):
+    subprocess.run(["ssh",f"lnls-bpm@{hostname}",command])
```

Comment:
You can make `command` hold all following [positional arguments as a list](https://docs.python.org/3/tutorial/controlflow.html#arbitrary-argument-lists) by defining it as:

```python3
def execute_ssh(hostname, *command):
     subprocess.run(["ssh", f"lnls-bpm@{hostname}", *command])
```

`*command` will [unpack it](https://docs.python.org/3/tutorial/controlflow.html#unpacking-argument-lists) inside the other list. This should remove the need to use formatted string on every call. Simply put them as a comma separated list of arguments.

BTW, I'm not sure if Python version we have in operation machines accepts f-strings. It should be [greater than or equal to 3.6](https://docs.python.org/3/reference/lexical_analysis.html#f-strings).

Reply:
In operation machines the Python version is 3.6.8, so it should be fine!